pkg/global/apply_configuration.go

```diff
@@ -24,33 +25,48 @@ import (
 // to all Buildbarn binaries, regardless of their purpose.
 func ApplyConfiguration(configuration *pb.Configuration) error {
 	// Push traces to Jaeger.
-	if jaegerConfiguration := configuration.GetJaeger(); jaegerConfiguration != nil {
-		pe, err := prometheus_exporter.NewExporter(prometheus_exporter.Options{
-			Registry:  prometheus.DefaultRegisterer.(*prometheus.Registry),
-			Namespace: "bb_storage",
-		})
-		if err != nil {
-			return util.StatusWrap(err, "Failed to create the Prometheus stats exporter")
-		}
-		view.RegisterExporter(pe)
-		if err := view.Register(ocgrpc.DefaultServerViews...); err != nil {
-			return util.StatusWrap(err, "Failed to register ocgrpc server views")
+	if tracingConfiguration := configuration.GetTracing(); tracingConfiguration != nil {
+		if jaegerConfiguration := tracingConfiguration.GetJaeger(); jaegerConfiguration != nil {
```

Comment:
Nit: It's perfectly fine to just use `.Jaeger` instead of `.GetJaeger()` here, as `tracingConfiguration` is guaranteed to be non-nill.